include/sdf/Link.hh

```diff
@@ -326,6 +326,14 @@ namespace sdf
     public: Errors ResolveInertial(gz::math::Inertiald &_inertial,
                                    const std::string &_resolveTo = "") const;
 
+    /// \brief Calculate & set inertial values(mass, mass matrix
+    /// & inertial pose) for the link. Inertial values can be provided
+    /// by the user through the SDF or can be calculated automatically
+    /// by setting the auto attribute to true.
+    /// \param[out] _errors A vector of Errors object. Each object
+    /// would contain an error code and an error message.
+    public: void CalculateInertials(sdf::Errors &_errors);
```

Comment:
Hmmm makes sense. Just to be sure that would involve renaming the `CalculateInertials()` for `Geometry` and all Geometry Types since they all return an optional `Inertiald` object? In that case would `ComputeInertials()` work? Or maybe just `Inertials()`?